instrumentation/cdi-testing/cdi-testing.gradle

```diff
@@ -1,4 +1,7 @@
-ext.skipPublish = true
+ext {
+  packageInAgentBundle = false
+  skipPublish = true
```

Comment:
This is great idea, but it bothers me, that one of these options is "positive" and another is "negative". Can we make them similar?

Reply:
We definitely should.

Reply:
Sorry didn't look closely - do `testing` libraries need to use instrumentation.gradle?

Reply:
It turns the flag is not needed at all, we can just add only projects that are named `javaagent` (thanks to our strict module naming convention)

Reply:
> Sorry didn't look closely - do `testing` libraries need to use instrumentation.gradle?

Some of the `testing` artifacts are not libraries, but modules that contain only tests for a particular environment - for example `glassfish-testing`, or those three classloader testing modules.